vendor/github.com/knative/test-infra/scripts/library.sh

```diff
@@ -146,6 +146,25 @@ function wait_until_pods_running() {
   return 1
 }
 
+# Waits until all pods are completed in the given namespace.
```

Comment:
I assume this is an experiment, and you actually don't plan to submit this PR as it is now, right?